docs/knowledge_base/tutorials/chatbot/index.md

```diff
@@ -0,0 +1,288 @@
+# Creating a Chatbot Website using Taipy
+
+In this tutorial we will create a simple chatbot website using Taipy.
+
+[Try it live](https://demo-llm-chat.taipy.cloud/){: .tp-btn target='blank' }
+[Get it on GitHub](https://github.com/Avaiga/demo-llm-chat){: .tp-btn .tp-btn--accent target='blank' }
+
+<p align="center">
+  <img src="chatbot_roundconv.png" alt="Render of the app" width="100%"/>
+</p>
+
+
+Here we will use OpenAI's API with GPT-3. This tutorial can easily
+ be adapted to other LLMs.
+
+## Table of Contents
```

Comment:
This can then be an H1. The same apply for all the subsequent titles

Reply:
Fixed